core/exception_vectors.S

```diff
@@ -491,7 +530,15 @@ NMIExceptionHandler:
         .balign  4
 UserExceptionHandler:
         .type   UserExceptionHandler, @function
-        xsr     a0, excsave1    # a0 now contains sp
+        // save a0, a1 to debug_saved_ctx before stack pointer is affected
```

Comment:
I'm probably missing something obvious, but why do we need to save a0 & a1 to debug_saved_ctx here? Is it because the 'break' instruction under LUserFailOtherExceptionCause triggers a DebugException? debug_saved_ctx gets written there as well, so I don't fully understand where these values are later used.